.github/workflows/test-lint.yml

```diff
@@ -33,3 +33,6 @@ jobs:
       - run: pnpm install
       - run: pnpm run lint
       - run: pnpm run prettier-check
+
+      # Enforce a changeset file to be present
+      - run: changeset status --since=main
```

Comment:
Note that this will only exit with non-zero if there's a code change in a package. Code changes outside of any package will still exit with `0`.

Reply:
<picture data-single-emoji=":ack:" title=":ack:"><img class="emoji" src="https://emoji.slack-edge.com/T0CAQ00TU/ack/fcab3a35263dfa53.png" alt=":ack:" width="20" height="auto" align="absmiddle"></picture> Probably ok I think?